File: WEB/Palladium/generateJSON.php
<?php
header('Access-Control-Allow-Origin: *');
header('Content-Type: appliction/json');

// *********************************************************************
$output = null;
$retval = null;
exec('bin/php_Image 142', $output, $retval);
$A[] = explode(';', $output[0]);

foreach ($A[0] as $a) {
	$B[] = explode("|", $a);
}

$len = sizeof($B) - 1;
unset ($B[$len]);

echo json_encode($B);

?>

File: WEB/Palladium/generateJSON.c
#include <stdio.h>

int main (int argc, char *argv[]) {
   printf(
      "Access-Control-Allow-Origin: *\n"
      "Content-Type: appliction/json\n\n"
      "[[\"A\",\"B\",\"C\"],\n"
      "[1,2,3],\n"
      "{\"name\":\"fish\"},\n"
      "[1,2,3,4]"
      "]\n"
   );
}
